filters/ThirdParty/filter_251_LegitimateURLShortener/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+  "filterId": 251,
+  "name": "Legitimate URL Shortener",
+  "description": "In a world dominated by bit.ly, ad.fly, and several thousand other malware cover-up tools, this list reduces the length of URLs in a much more legitimate and transparent manner. Essentially, it automatically removes unnecessary $/& values from the URLs, making them easier to copy from the URL bar and pasting elsewhere as links.",
+  "timeAdded": 1626086804599,
+  "homepage": "https://github.com/DandelionSprout/adfilt",
+  "expires": "4 days",
+  "displayNumber": 251,
```

Comment:
this parameter will make this filter the last in privacy group. are you sure it's should be after fanboy's filters?

Reply:
Well, I were thinking about it. In the end, this is a safe option, but if it is in front of `EasyPrivacy`, then next to our `AdGuard URL Tracking filter`.